changelog.md

```diff
@@ -1,3 +1,6 @@
+### v10.7.2
```

Comment:
Please check if this will be the right version as Przemek plans to merge #609 (10.8.0)